code/modules/surgery/organs/internal/cyberimp/augments_chest.dm

```diff
@@ -52,13 +52,16 @@
 	slot = ORGAN_SLOT_HEART_AID
 	var/revive_cost = 0
 	var/reviving = FALSE
+	var/can_defib_owner
 	COOLDOWN_DECLARE(reviver_cooldown)
 
+/obj/item/organ/internal/cyberimp/chest/reviver/on_death(seconds_per_tick, times_fired)
+	on_life() // Allowes implant to work even on dead people
```

Comment:
Not a huge fan of this, I'd rather `on_life` and `on_death` both call a proc like `try_heal` which has the code that either calls heal or starts the cd

Reply:
okay